mordant/src/commonMain/kotlin/com/github/ajalt/mordant/terminal/TerminalDetection.kt

```diff
@@ -4,13 +4,13 @@ import com.github.ajalt.mordant.internal.*
 import com.github.ajalt.mordant.rendering.AnsiLevel
 import com.github.ajalt.mordant.rendering.AnsiLevel.*
 
-internal object TerminalDetection {
+object TerminalDetection {
```

Comment:
I don't think there's any reason to make this public. Instead of `TerminalDetection.detectTerminal()`, folks can can the same object from `Terminal().info`.

Reply:
Thanks for pointing out `Terminal().info`, that's much nicer, and I've dropped this commit from the PR.